pages/index.js

```diff
@@ -245,24 +293,42 @@ const ShoppingList = () => {
         <Button type="submit">Add Item</Button>
       </FormContainer>
 
-      <p>Total items: {totalItems}</p>
-      <div className="items-container">
-        {items.map((item) => (
-          <Link key={item.id} href={`/item/${item.id}`} passHref>
-            <ItemCard category={item.category}>
-              <h3>{item.name}</h3>
-              <p>Quantity: {item.quantity}</p>
-              <p>
-                Category:{" "}
-                {categories.find(
-                  (cat) => cat.toLowerCase() === item.category
-                ) || item.category}
-              </p>
-              {item.comment && <p>Comment: {item.comment}</p>}
-            </ItemCard>
-          </Link>
-        ))}
-      </div>
+      {totalItems === 0 ? (
+        <div>No shopping items. </div>
```

Comment:
I´d give the user a hint to add some shopping items by clicking on "add items". That would be a GAMECHANGER 😂